packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -282,6 +283,22 @@ storage:
           kind: KubeletConfiguration
           cgroupDriver: "$${docker_cgroup_driver}"
           EOF
+    - path: /opt/etcd-private-interface
+      filesystem: root
+      mode: 0744
+      contents:
+        inline: |
+          #!/bin/bash
+          set -e
+          if test -f /tmp/etcd-private-interface; then
```

Comment:
How about `grep REPLACE_WITH_PRIVATE_IP /etc/systemd/system/etcd-member.service.d/40-etcd-cluster.conf` instead? With file in `/tmp`, the script will re-run on every boot. I don't think private IP changes on Packet over time.

Reply:
done